bootstrap/kubeadm/internal/cloudinit/kubeadm-bootstrap-script.sh

```diff
@@ -117,13 +122,17 @@ function try-or-die-command() {
 }
 
 retry-command kubeadm join phase preflight
-retry-command kubeadm join phase control-plane-prepare download-certs
-retry-command kubeadm join phase control-plane-prepare certs
-retry-command kubeadm join phase control-plane-prepare kubeconfig
-retry-command kubeadm join phase control-plane-prepare control-plane
+if [ "$OPTION" == "controlplane" ]; then
```

Comment:
Instead of this if condition in the file itself, could we make use of the go template conditionals to only render what we need?

Reply:
This would be ideal to avoid hitting limits when passing this through cloud-init (CAPV has a 64k limitation on `guestinfo`)

Reply:
Adding